tools/cd_scripts/install_test.sh

```diff
@@ -20,6 +20,7 @@ set -x
 gsutil cp  gs://gcsfuse-release-packages/version-detail/details.txt .
 # Writing VM instance name to details.txt (Format: release-test-<os-name>)
 vm_instance_name=$(curl http://metadata.google.internal/computeMetadata/v1/instance/name -H "Metadata-Flavor: Google")
+to_release_version=$(sed '1q' details.txt | tr -d '\n')
```

Comment:
The release version in details.txt is in the format vX.Y.Z, for example, v2.2.0. If this is the case, shouldn't we only capture 2.2.0 as the to_release_version? If not, could you please provide a comment indicating the expected format of the first line in details.txt.